yarn.lock

```diff
@@ -1209,6 +1197,20 @@
     ws "^7.5.9"
     yaml "^2.3.4"
 
+"@forge/runtime@5.10.3":
```

Comment:
If it were possible to upgrade other dependencies that were asking for forge/runtime version 5.10.1 to also use 5.10.3, that might be helpful, so that the whole application would only depend on a specific runtime version.